src/Codeception/Subscriber/Console.php

```diff
@@ -526,8 +526,13 @@ protected function detectWidth()
             && (getenv('TERM') != 'unknown')
         ) {
             $this->width = (int) (`command -v tput >> /dev/null 2>&1 && tput cols`) - 2;
+        } elseif ($this->isWin() && (php_sapi_name() == "cli")) {
```

Comment:
yes, true
should have been used in the above if linux test too, why wasn't it used?